operators/src/processing/reprojection.rs

```diff
@@ -416,10 +416,20 @@ where
     ) -> Result<BoxStream<'a, Result<geoengine_datatypes::raster::RasterTile2D<Self::RasterType>>>>
     {
         // we need a resolution for the sub-querys. And since we don't want this to change for tiles, we precompute it for the complete bbox and pass it to the sub-query spec.
-        // TODO: use `rewrite_query` to determine resolution for tiles overlapping border
-        let projector = CoordinateProjector::from_known_srs(self.to, self.from)?;
+        let projector_source_target = CoordinateProjector::from_known_srs(self.from, self.to)?;
+        let projector_target_source = CoordinateProjector::from_known_srs(self.to, self.from)?;
+
+        let p_bbox = query.bbox.reproject_clipped(&projector_target_source)?;
+        let s_bbox = p_bbox.reproject(&projector_source_target)?;
```

Comment:
I do exactly the same as for vector queries. I just can't use the `rewrite_query` function because for rasters the `RasterOverlapAdapter` does the actual rewriting.